### PR TITLE
[lambda] Add `DD_CAPTURE_LAMBDA_PAYLOAD` support.

### PR DESCRIPTION
**Pull request by duncanista**
### What and why?

`DD_CAPTURE_LAMBDA_PAYLOAD` is the flag which controls whether we capture and transmit lambda payloads to the backend. 
It will be set with the option `--capture-lambda-payload` and with the config variable `captureLambdaPayload`. They need to be set with a boolean or a quoted boolean (string boolean, being a boolean between quotes such as `'false'|"FALSE"`, etc)

Refer to [SLS-1546](https://datadoghq.atlassian.net/browse/SLS-1546).

### How?

- Adding constants for the environment variable. 
- Updated tests.

### Review checklist

- [x] Feature or bugfix MUST have appropriate tests (unit, integration)

### CODE REVIEW

File: src/commands/lambda/__tests__/functions/instrument.test.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ describe('instrument', () => {
             "Variables": Object {
               "DD_FLUSH_TO_LOG": "false",
               "DD_LAMBDA_HANDLER": "handler.hello",
-              "DD_MERGE_XRAY_TRACES": "false",
```
Comment (IvanTopolcic) [inline src/commands/lambda/__tests__/functions/instrument.test.ts:102]: Why the changes to these tests?

**Review by IvanTopolcic — COMMENTED** [review]
Looks good to me, just a quick question!

File: src/commands/lambda/__tests__/functions/instrument.test.ts (hunk 1 above)
Comment (duncanista) [reply src/commands/lambda/__tests__/functions/instrument.test.ts:102]: I reverted a change and forgot to revert the tests too. Will change them back, thank you for pointing it out.

**Review by IvanTopolcic — APPROVED** [review]
LGTM!
